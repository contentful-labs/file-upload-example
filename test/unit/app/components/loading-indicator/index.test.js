import { h } from 'preact'
import render from 'preact-render-to-string'

import LoadingIndicator from '../../../../../src/app/components/loading-indicator/index'

test('LoadingIndicator', () => {
  const props = {
    message: 'Pretty busy loading'
  }
  const html = render(<LoadingIndicator {...props} />)
  expect(html).toMatchSnapshot()
})
