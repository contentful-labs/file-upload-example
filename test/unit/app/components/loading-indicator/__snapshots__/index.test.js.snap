// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator 1`] = `
<div>
  <div class="wrapper">
    <div class="content">
      <div class="logo">
        <img src="test-file-stub" alt="Contentful" />
      </div>
      <div class="message">Pretty busy loading</div>
    </div>
  </div>
</div>
`;
